src/debug.h

```diff
@@ -10,6 +10,24 @@
 #ifndef _CH8_SRC_DEBUG_H_
 #define _CH8_SRC_DEBUG_H_
 
+#include "cli.h"
 
+typedef enum {
+    debug,
+    info,
+    warning,
+    error,
+    fatal
+} debug_level_t;
```

Comment:
I think that using caps for the enum members would be more appropriate.

Reply:
:+1: